src/dndbeyond/base/monster.js

```diff
@@ -489,6 +491,13 @@ class Monster extends CharacterBase {
                 damage_types.push(dmg[4]);
             }
         }
+        for (let dmg of healing_matches) {
+            const damage = dmg[2] || dmg[1];
+            if (damage) {
+                damages.push(damage.replace("plus", "+"));
+                damage_types.push("Healing");
```

Comment:
Make the `(?:temporary)?` into a capturing group and check for `dmg[3]` value, so if it's temp hp, the damage type should be "Temp HP" (It's what we used elsewhere, see `src/dndbeyond/content-scripts/charater.js` line 1504)

Reply:
Thanks for the guidance, I've made the update.